webapp/src/main/resources/public/js/ict/IctModalContainer.js

```diff
@@ -43,13 +57,33 @@ class IctModalContainer extends React.Component {
         window.open(this.state.mojitoBaseUrl + "workbench?" + query);
     }
 
+    createGlobalExperiment() {
+        var query = queryString.stringify({
+            'repoName': this.state.selectedTextUnit.repositoryName,
+            'searchText': this.state.selectedTextUnit.textUnitName
+        });
+        window.open(this.state.experimentUrl + this.state.experimentButtons['global']['urlCallback'] + '&' + query);
+    }
+
+    createLocalExperiment() {
+        var query = queryString.stringify({
```

Comment:
so I guess in the generic version will end up passing an extra parameter for global but that won't be a big deal